pkg/logs/message/origin.go

```diff
@@ -16,21 +16,25 @@ type Origin struct {
 	Identifier string
 	LogSource  *config.LogSource
 	Offset     string
+	Service    string
+	Source     string
```

Comment:
it's confusing to have Source and LogSource. Maybe we should rename LogSource into LogConfig?